docker/next-app/Dockerfile.next-app

```diff
@@ -0,0 +1,22 @@
+# Build the app
+FROM node:alpine AS builder
+
+WORKDIR /app
+
+COPY jsconfig.json package-lock.json package.json postcss.config.cjs svelte.config.js tailwind.config.cjs /app
+COPY src /app/src
+COPY static /app/static
+
+RUN npm install
```

Comment:
after this PR is merged, we can move npm install out of docker

Reply:
I don't think so...we publish this image and then utilize it in our runtime so everything needs to be in the image.

Reply:
this is just a comment really.  I think we need to discuss our npm and Docker strategy more so we're on the same page.